bob/djid/column/__init__.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+"""Column definitions."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.db.models import (
+    Count,
+    Field,
+    ForeignKey,
+    CharField,
+    DateField,
+    DateTimeField,
+    DecimalField,
+    IntegerField,
+)
+
+from bob.djid.util import PEP3115
+
+
+if not PEP3115:
+    _counter = 0
+
+
+class Column(object):
+    """A column object.
+        :param as_link: True or False to force creating links to related object
```

Comment:
what doc string style is it? especially that a different style is used below. Nice guidelines are used by https://github.com/openstack-dev/hacking/blob/master/hacking/checks/docstrings.py